### ISSUE TITLE
Spurious "Create methods should include `parent`" warning

### ISSUE THREAD

**Issue by drigz** [googleapis/api-linter#408]
Service:

```
  rpc CreateProject(CreateProjectRequest) returns (Project) {
    option (google.api.http) = {
      post: "/v1alpha1/{parent=organizations/*}/projects"
      body: "project"
    };
  }
```

Warning:

```
  - message: Create methods should include the `parent` field in the URI.
    location:
      start_position:
        line_number: 52
        column_number: 3
      end_position:
        line_number: 57
        column_number: 3
    rule_id: core::0133::http-uri-parent
    rule_doc_uri: https://linter.aip.dev/133/http-uri-parent
```

I can't work out what I'm missing here as the my service seems identical to the example from the docs.

**Comment by lukesneeringer** [comment]
We did have a false positive on this, which I believe we fixed in #402. I assume you are using the linter internally at Google; can you sync the CL and tell me if this is still an issue?

**Comment by drigz** [comment]
I can confirm the fix, thanks Luke!
